resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java

```diff
@@ -1836,16 +1835,16 @@ private List<String> convertServiceTypeToResidentTransactionType(String serviceT
 				if (service.equalsIgnoreCase(ServiceType.AUTHENTICATION_REQUEST.toString())) {
```

Comment:
No need of if else condition, you can just pass the ServiceType obtained using `service` and pass it.

Reply:
changes done